### PR TITLE
Feature/yarn

### PR DESCRIPTION
**Pull request by st3v3nhunt**
Add the 'standard' scripts too

### CODE REVIEW

File: scripts/start [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+docker-compose down -v
+docker-compose up --build --force-recreate
```
Comment (markysoft) [inline scripts/start]: missing profiles-frontend on the up command

**Review by markysoft — CHANGES_REQUESTED** [review]
The start command is not working for me.
I get an error:
 ./node_modules/.bin/bunyan: not found

The start command is running three containers, rather than just the required two.

**Comment by st3v3nhunt** [comment]
@markysoft Yeah sorry about that. Pretty stupid thing to overlook. Not sure what the fix for bunyan is. I'll have a look at it.

I also need to review the main `README` and make sure it points to the one for the scripts.

**Comment by st3v3nhunt** [comment]
@markysoft Should be good to go now:

* The error with Bunyan is sorted (the binary wasn't in the same place since the move to yarn)
* The specific service required to run the app has been specified
* The readme has been updated and points to the scripts

**Review by markysoft — APPROVED** [review]

**Pull request merged**